operator/cmd/flags.go

```diff
@@ -287,13 +287,17 @@ func init() {
 	option.BindEnv(Vp, option.EnableCiliumEndpointSlice)
 
 	flags.Int(operatorOption.CESMaxCEPsInCES, operatorOption.CESMaxCEPsInCESDefault, "Maximum number of CiliumEndpoints allowed in a CES")
-	flags.MarkHidden(operatorOption.CESMaxCEPsInCES)
 	option.BindEnv(Vp, operatorOption.CESMaxCEPsInCES)
 
 	flags.String(operatorOption.CESSlicingMode, operatorOption.CESSlicingModeDefault, "Slicing mode define how ceps are grouped into a CES")
-	flags.MarkHidden(operatorOption.CESSlicingMode)
 	option.BindEnv(Vp, operatorOption.CESSlicingMode)
 
+	flags.Float64(operatorOption.CESWriteQPSLimit, operatorOption.CESWriteQPSLimitDefault, "CES work queue rate limit")
```

Comment:
I think in the code we should set a limit on this, same as below, otherwise an arbitrary number form user might break their cluster easily.

Reply:
Thank you @Weil0ng for the review and suggestion.

You are right. Most users wouldn’t know how to set CES flags to their benefits.
I think it’s anyway good to show the flags when `EnableCiliumEndpointSlice` is turned on, as it might benefit users who are willing to inquire and learn about it.
Do you think it’s a good idea to show the related flags only when CES is enabled?

Users could have misconfigured CES processing/write rate anyway with `K8sClientQPS` in operator.
And I think it isn't easy to break a cluster by misconfiguring it, because the cluster needs to run at scale in the first place (many nodes, high pod churn rate, many pods in total), and that’s rare.

I anyway agree that we should add a hard limit, but I’m not sure which number it should be.
We observed that it could be 100+ QPS for smaller clusters, but 10 QPS for very large clusters, so even with a hard limit it’s possible that users could break their clusters.
Do you have a suggestion?

Reply:
Thank @dlapcevic ! I agree that it is hard (if possible) to pick a limit that suits all cases, maybe a better approach is to have a step function that limit the QPS based on number of nodes in the cluster, but I don't think it should block this PR anyways :)

Reply:
nit: since we added a hard limit, maybe it's worthing mentioning in the description text for the flag that it cannot exceed X?

Reply:
Done.